.envrc

```diff
@@ -1,5 +1,6 @@
 export PAK_BUILD_CACHE_PATH=/tmp/.pak-cache
 export GSB_COMPATIBILITY_ENABLE_BETA_SERVICES=true
+export CSB_DISABLE_TF_UPGRADE_PROVIDER_RENAMES=true
```

Comment:
Oh, I see now that this line comes from cherry-picking the commit from PR #881

It was a little bit confusing. I don't know how this would play out if the PRs were merged in the wrong order or were "stashed and merged" though.

I guess you will be taking care of merging anyway, so that shouldn't be a problem. :)

Reply:
Yes, that's why I left this one as draft for the time being.